ports/tensorflow/portfile.cmake

```
@@ -0,0 +1,16 @@
+set(TF_LIB_SUFFIX "")
```

```suggestion
vcpkg_fail_port_install(ON_ARCH "x86" "arm" ON_TARGET "uwp")

set(TF_LIB_SUFFIX "")
```
It would be better to add this unsupported messages here.